### PR TITLE
[hotfix][docs] fix config.zh.md file path error

### PR DESCRIPTION
**Pull request by sunjincheng121**
Just hotfix `config.zh.md` file path error.

### CODE REVIEW

**Comment by sunjincheng121** [comment]
Keep consist with `config.md` see： https://github.com/apache/flink/blob/master/docs/ops/config.md Line 111.

File: docs/ops/config.zh.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The default fraction for managed memory can be adjusted using the taskmanager.me
 
 These parameters allow for advanced tuning. The default values are sufficient when running concurrent high-throughput jobs on a large cluster.
 
-{% include generated/netty_configuration.html %}
+{% include generated/network_netty_configuration.html %}
```
Comment (zhijiangW) [inline docs/ops/config.zh.md]: It should also include this 

### Network Environment

{% include generated/network_environment_configuration.html %}

Comment (sunjincheng121) [reply docs/ops/config.zh.md]: This patch fixes the file path error, but I am fine to add you mention section to align with the config.

**Comment by wuchong** [comment]
LGTM

**Comment by sunjincheng121** [comment]
Thanks for your review @zhijiangW @wuchong!
I have adds the `network environment` section to align the `config.md`.
Best,
Jincheng

**Comment by sunjincheng121** [comment]
Merging...

**Pull request closed without merging**